shared/src/packages/journal/libs/types/journal-entry-create-response-dto.type.ts

```diff
@@ -0,0 +1,7 @@
+import { type JournalEntryGetAllItemResponseDto } from './journal-entry-get-all-item-response-dto.type.js';
+
+type JournalEntryCreateResponseDto = {
+  journalEntry: JournalEntryGetAllItemResponseDto;
+};
```

Comment:
Why not to use the same type, is it supposed to have some differences in future? You just wrapped it in curly brackets, that made some troubles in backend and frontend communication, at least make this type equal to `JournalEntryGetAllItemResponseDto` in order to save namings

Reply:
done!